include/oneapi/dpl/pstl/hetero/dpcpp/unseq_backend_sycl.h

```diff
@@ -35,6 +35,48 @@ using void_type = typename ::std::enable_if<::std::is_void<_Tp>::value, _Tp>::ty
 template <typename _Tp>
 using non_void_type = typename ::std::enable_if<!::std::is_void<_Tp>::value, _Tp>::type;
 
+#if _USE_GROUP_ALGOS
+//TODO: To change __has_known_identity implementation as soon as the DPC++ compiler implementation issues related to
+//std::multiplies, std::bit_or, std::bit_and and std::bit_xor operations will be fixed.
+//std::logical_and and std::logical_or are not supported in DPC++ compiler to be used in sycl::inclusive_scan_over_group and sycl::reduce_over_group
+template <typename BinaryOp, typename _Tp>
+using __has_known_identity =
+#    if __LIBSYCL_VERSION >= 50200
+    typename ::std::conjunction<
+        ::std::is_arithmetic<_Tp>, sycl::has_known_identity<BinaryOp, _Tp>,
+        ::std::disjunction<::std::is_same<typename ::std::decay<BinaryOp>::type, ::std::plus<_Tp>>,
+                           ::std::is_same<typename ::std::decay<BinaryOp>::type, sycl::plus<_Tp>>,
+                           ::std::is_same<typename ::std::decay<BinaryOp>::type, sycl::minimum<_Tp>>,
+                           ::std::is_same<typename ::std::decay<BinaryOp>::type, sycl::maximum<_Tp>>>>;
```

Comment:
In oneDPL, we have added our own implementations of `minimum` and `maximum` which are almost identical to SYCL definitions. If these oneDPL functors are passed in, can we somehow replace those with SYCL analogues, in order to use group reductions?